packages/@atjson/renderer-commonmark/test/line-break.test.ts

```diff
@@ -0,0 +1,201 @@
+import CommonmarkRenderer from "../src";
+
+describe("terminal line breaks are removed", () => {
+  test("in text", () => {
+    const document = {
+      text: "\ufffctest\ufffc",
+      blocks: [
+        {
+          id: "B00000000",
+          type: "text",
+          parents: [],
+          selfClosing: false,
+          attributes: {},
+        },
+        {
+          id: "B00000000",
+          type: "line-break",
+          parents: ["text"],
+          selfClosing: true,
+          attributes: {},
+        },
+      ],
+      marks: [],
+    };
+
+    expect(CommonmarkRenderer.render(document)).toBe("test");
+  });
+
+  test("in text with preceding linebreaks", () => {
+    const document = {
+      text: "\ufffctest\ufffctest\ufffc",
+      blocks: [
+        {
+          id: "B00000000",
+          type: "text",
+          parents: [],
+          selfClosing: false,
+          attributes: {},
+        },
+        {
+          id: "B00000000",
+          type: "line-break",
+          parents: ["text"],
+          selfClosing: true,
+          attributes: {},
+        },
+        {
+          id: "B00000000",
+          type: "line-break",
+          parents: ["text"],
+          selfClosing: true,
+          attributes: {},
+        },
+      ],
+      marks: [],
+    };
+
+    expect(CommonmarkRenderer.render(document)).toBe("test\\\ntest");
+  });
+
+  test.skip("in text terminating a mark", () => {
```

Comment:
This test fails, but is at least consistent with the existing behavior we have with terminal linebreaks inside of a mark at the end of a block.